test.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const greet =  require('./lib/greet.js');
+const assert = require('assert');
+describe('greet', function(){
+   describe('#sayHi', function(){
+    it('should return Hello Bhavya!', function(){
+     var call = greet.sayHi('Bhavya');
+     assert.ok(call === 'Hello Bhavya!', 'not equal Hello Bhavya!');
+    })
+   });
+});
```

Comment:
Minor, minor nitpick- indentation on lines 10-11 and missing semi-colon on line 10 :)